connectors/migrations/20240828_microsoft_refill_parents_field.ts

```diff
@@ -0,0 +1,74 @@
+import { makeScript } from "scripts/helpers";
+import { Op } from "sequelize";
+
+import { getParents } from "@connectors/connectors/microsoft/temporal/file";
+import { concurrentExecutor } from "@connectors/lib/async_utils";
+import { updateDocumentParentsField } from "@connectors/lib/data_sources";
+import { MicrosoftNodeModel } from "@connectors/lib/models/microsoft";
+import logger from "@connectors/logger/logger";
+import { ConnectorResource } from "@connectors/resources/connector_resource";
+
+makeScript({}, async ({ execute }) => {
+  const connectors = await ConnectorResource.listByType("microsoft", {});
+  for (const connector of connectors) {
+    logger.info({ connector: connector.toJSON() }, "Processing connector");
+    await updateParentsFieldForConnector(connector, execute);
+  }
+});
+
+async function updateParentsFieldForConnector(
+  connector: ConnectorResource,
+  execute = false
+) {
+  // get all nodes (not using resource here since 1-shot clean migration)
+  const nodes = await MicrosoftNodeModel.findAll({
```

Comment:
We need to also get tables, and update the parents list with `updateTableParentsField`

Reply:
ok looks we don't have the issue with tables, no issue here